### PR TITLE
PS-6890 Documented the new variable blind_delete_primary_key

### PR DESCRIPTION
**Pull request by patrickbirch**
modified myrocks/variable
added myrocks_blind_delete_primary_key and description

### CODE REVIEW

**Review by inikep — CHANGES_REQUESTED** [review]
Scope is `Global, Session`

**Comment by patrickbirch** [comment]
updated to Global, Session

File: doc/source/myrocks/variables.rst [hunk 1]
```diff
@@ -64,6 +64,10 @@ Also, all variables can exist in one or both of the following scopes:
      - Yes
      - No
      - Global
+   * - :variable:`rocksdb_blind_delete_primary_key`
+     - Yes
+     - Yes
+     - Global
```
Comment (inikep) [inline doc/source/myrocks/variables.rst]: Global, Session

**Review by inikep — CHANGES_REQUESTED** [review]

File: doc/source/myrocks/variables.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/myrocks/variables.rst]: Missed that. Fixed.

**Review by inikep — APPROVED** [review]

**Comment by inikep** [comment]
LGTM

**Pull request merged**